cmd/minikube/cmd/status.go

```diff
@@ -544,7 +547,7 @@ func clusterState(sts []*Status) ClusterState {
 			}
 			exitCode, err := strconv.Atoi(data["exitcode"])
 			if err != nil {
-				klog.Errorf("unable to convert exit code to int: %v", err)
```

Comment:
I think this should still be emitted to stderr, as it's a non-fatal programming flaw that we should address rather than hide.

Reply:
agree